src/index.html

```diff
@@ -7,13 +7,14 @@
     <title>Document</title>
   </head>
   <body>
-    <div class="container">
-        <h1>Leaderboard</h1>
+    <div class="main">
+      <h1>Leaderboard</h1>
+      <div class="container">
         <div class="content">
           <div class="col-1">
             <span class="one-line-row">
               <h2 class="mr-1">Recent Scores</h2>
-              <button class="btn-refresh">Refresh</button>
+              <button class="btn-refresh btn">Refresh</button>
             </span>
             <div class="recent-scores"></div>
           </div>
```

Comment:
- [ ] Beautiful design you do have 👏 . However, kindly adjust the score board to either be closely aligned with the `Leader-board` header
- Also, consider reducing large gap between the form and the score board. This is to give  a better visual appealing view 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/231679303-422f1c23-525d-4d4c-8814-2af74808f825.png)

## Wireframe:
![image](https://user-images.githubusercontent.com/50990925/231679446-1ad918b9-749b-44dd-aa4c-405718002271.png)